_includes/master/selectors.md

```diff
@@ -2,9 +2,12 @@ A label selector is an expression which either matches or does not match an endp
 
 {{site.prodname}} label selectors support a number of syntactic primitives.  Each of the following 
 primitive expressions can be combined using the logical operator `&&`.
+If any selector is used in a Namespaced context then it is limited to match
```

Comment:
I'd probably make this statement more concrete - for example:

"selectors on NetworkPolicy resources select only objects that are in the same namespace as the policy. Selectors on GlobalNetworkPolicy objects select across all namespaces"

Main thing is there is no definition of "in a namespaced context"

Reply:
Scratch that, I see what you did with the definition of namespace context above.

Reply:
Still feel like this sentence is confusing... but not sure how to make it more clear.